### PR TITLE
pr-check: Color test output

### PR DESCRIPTION
**Pull request by 2uasimojo**
...so it can be visually distinguished from the output of the commands run by the tests. This will make debugging easier.

### CODE REVIEW

**Comment by 2uasimojo** [comment]
Here's a snippet of what the logs look like with this change (got this from `oc logs` from a [CI run](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_boilerplate/109/pull-ci-openshift-boilerplate-master-pr-check/1347663277135499264)):
Screenshot from 2021-01-08 16-26-27

File: test/lib.sh [hunk 1]
```diff
@@ -18,10 +18,18 @@ export NEXUS_MK=boilerplate/generated-includes.mk
 _BP_TEST_TEMP_DIRS=
 
 err() {
-  echo "==ERROR== $@" >&2
+  echo -e "\e[31m==ERROR== $@\e[0m" >&2
```
Comment (dustman9000) [inline test/lib.sh:21]: I'd suggest defining these ANSI colors to a variable so we know what color this is.

File: test/lib.sh [hunk 2]
```diff
@@ -18,10 +18,18 @@ export NEXUS_MK=boilerplate/generated-includes.mk
...
   exit 1
 }
 
+# Override "echo" so test output is visually distinguished.
+# NOTE: This means when you need to echo something raw (e.g. when
+# constructing a file, or as "output" from a function), you need to use
+# /bin/echo explicitly.
+echo() {
+  /bin/echo -e "\e[33m$@\e[0m"
```
Comment (dustman9000) [inline test/lib.sh:30]: Same here, define YELLOW and RESET ...

File: test/lib.sh (hunk 1 above)
Comment (2uasimojo) [reply test/lib.sh:21]: Yeah, there's actually a way to use some kind of color name here instead. I'll look into it.

**Pull request merged**